corehq/apps/analytics/tasks.py

```diff
@@ -858,5 +861,14 @@ def cleanup_blocked_hubspot_contacts():
     if not HUBSPOT_ENABLED:
         return
 
+    time_started = datetime.utcnow()
+
     remove_blocked_domain_contacts_from_hubspot()
     remove_blocked_domain_invited_users_from_hubspot()
+
+    task_time = datetime.utcnow() - time_started
+    metrics_gauge(
+        'commcare.hubspot.runtimes.cleanup_blocked_hubspot_contacts',
+        task_time.seconds,
```

Comment:
Maybe a gauge is definitely what you want here, and if so ignore the rest of this comment. `metrics_histogram` might work well here. Example: https://github.com/dimagi/commcare-hq/blob/f12a66af7d335121b17f21b7f4e1691609a77be7/corehq/ex-submodules/casexml/apps/phone/restore.py#L767-L771

Reply:
I'm following the example in the HQ > Hubspot periodic task, which also uses metrics gauge and produces the graph I want on Datadog.

https://github.com/dimagi/commcare-hq/blob/a7923f5a5300feb423cd7ee22d780e5a4bcc8e87/corehq/apps/analytics/tasks.py#L642-L646